src/Orleans.sln

```diff
@@ -229,6 +229,16 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "BondUtils.Tests", "..\test\
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Consul.Tests", "..\test\Consul.Tests\Consul.Tests.csproj", "{D964FE9E-008A-4EDB-BBD7-8C57185C7A18}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Versions", "Versions", "{2C188181-1B01-4E77-A83D-F6A2EA855BE9}"
```

Comment:
I combined interfaces with their grain implementations. Even if I could merge V1 in the TestGrains, I prefer to keep two projects for grains because it will enable more testing scenario later (deploy v2, then deploy v1)